onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -1536,9 +1536,10 @@ TEST(ReductionOpTest, ReduceSumBFloat16) {
 
 // on CUDA - this UT, with axes {0,2}, will go thru cudnn lib only if ATenOp is not initialized
 // on ROCM - miopen call succeeded, but results in data error, thus follow the same logic done in cudnn for now
+//           4.2 doesn't run properly, thus enable the UT only above 4.3
 // TODO - try ROCm 4.5.2 and/or double check the source code on BFloat16 support 
-#if defined(USE_CUDA) || defined(USE_ROCM)
-TEST(ReductionOpTest, DISABLED_ReduceSumBFloat16_2) {
+#if defined(USE_CUDA) || (defined(USE_ROCM)  && ROCM_VERSION >= 40300)
```

Comment:
What's the problem when using 4.2?

Reply:
The calculation is way off, rel_error = 1 mean it's 100% different - likely means the returned value is 0 (expected to be 33, and 45). We can look deeper if we want. But given our priority is ROCm 4.5, not sure we want to spend more time on ROC m 4.2. Thought?

[ RUN      ] ReductionOpTest.ReduceSumBFloat16_2
/onnxruntime_src/onnxruntime/test/providers/provider_test_utils.cc:345: Failure
The difference between 0 and rel_error is 1, which exceeds threshold, where
0 evaluates to 0,
rel_error evaluates to 1, and
threshold evaluates to 0.05000000074505806.
provider_type: ROCMExecutionProvider:
/onnxruntime_src/onnxruntime/test/providers/provider_test_utils.cc:345: Failure
The difference between 0 and rel_error is 1, which exceeds threshold, where
0 evaluates to 0,
rel_error evaluates to 1, and
threshold evaluates to 0.05000000074505806.
provider_type: ROCMExecutionProvider:
[  FAILED  ] ReductionOpTest.ReduceSumBFloat16_2 (31 ms)